spec/interactors/datapass_webhook/archive_current_token_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe DatapassWebhook::ArchiveCurrentToken, type: :interactor do
+  subject { described_class.call(datapass_webhook_params.merge(authorization_request:)) }
+
+  let(:token) { create(:token) }
+
+  let(:authorization_request) { create(:authorization_request, tokens: [token]) }
+  let(:datapass_webhook_params) { build(:datapass_webhook, event:) }
+
+  context 'when event is not archive' do
```

Comment:
je chipote un peu mais:


```suggestion
  context "when event is not 'archive'" do
```